### ISSUE TITLE
Add Note about Click Indications to Instructor Notes

### ISSUE THREAD

**Issue by colinquirk** [datacarpentry/OpenRefine-ecology-lesson#230]
It may be hard to follow where the mouse is on the screen and when items are clicked when working with GUI programs. Some instructors may wish to use a program to highlight their mouse and indicate when they click. This is often seen when watching a recording of a screen, but the same concept may work live as well.

https://superuser.com/questions/193849/how-do-i-graphically-pop-keystrokes-and-mouse-clicks-so-they-are-useful-in-pla

**Comment by villanueval** [comment]
This will be specific to the setting, so closing since it would be too broad and outside scope to try to recommend a simple solution.
